main.py

```diff
@@ -1,10 +1,38 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        players = json.load(players)
+
+    for p_name in players:
+        player = players[p_name]
+        race, _ = Race.objects.get_or_create(
+            name=player["race"]["name"],
+            description=player["race"]["description"])
+
+        for skill_data in player["race"]["skills"]:
+            skill, _ = Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"], race=race)
+
+        if player["guild"] is not None:
```

Comment:
You could simplify this check
```suggestion
        if player.get("guild"):
```